src/Monitor/Monitor/help/Get-AzScheduledQueryRule.md

```diff
@@ -0,0 +1,139 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll-Help.xml
+Module Name: Az.Monitor
+online version: https://docs.microsoft.com/en-us/powershell/module/az.monitor/get-azscheduledqueryrule
+schema: 2.0.0
+---
+
+# Get-AzScheduledQueryRule
+
+## SYNOPSIS
+Gets Scheduled Query Resources
+
+## SYNTAX
+
+### BySubscriptionOrResourceGroup (Default)
+```
+Get-AzScheduledQueryRule [-ResourceGroupName <String>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### ByRuleName
+```
+Get-AzScheduledQueryRule -ResourceGroupName <String> -Name <String> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### ByResourceId
+```
+Get-AzScheduledQueryRule -ResourceId <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Gets Scheduled Query Resources
+
+## EXAMPLES
+
+### Example 1 - List by subscription or resource group
+```powershell
+PS C:\> Get-AzScheduledQueryRule -ResourceGroupName "Rac46PostSwapRG"
```

Comment:
@shpimpal for your examples, you should have the corresponding output from the cmdlet being executed in the code block so users know what the command will return

Reply:
Done